components/feedstock/stores.js

```diff
@@ -87,39 +100,56 @@ const Store = ({ dataset }) => {
           <Code language='python'>{getSnippet(url)}</Code>
 
           <Button
-            as={Link}
-            href={`https://data-viewer-git-katamartin-pyramid-maps-carbonplan.vercel.app/?dataset=${pyramid || url}`}
+            href={`https://ncview-js.staging.carbonplan.org/?dataset=${
+              pyramid || url
+            }`}
             target='_blank'
             rel='noopener noreferrer'
-            suffix={<RotatingArrow sx={{ ml: '2px' }} />}
+            onClick={(e) => {
+              if (!dataset.public) {
+                e.preventDefault() // Prevents the link from being followed
+              }
+            }}
+            suffix={
+              <RotatingArrow
+                sx={{
+                  transform: ['translateY(-6%)'],
+                }}
+              />
+            }
             sx={{
+              color: color,
+              fontSize: [0, 0, 0, 1],
+              letterSpacing: 'mono',
+              textTransform: 'uppercase',
+              fontFamily: 'mono',
               mt: 3,
               mb: 2,
-              fontSize: 0,
-              fontFamily: 'mono',
-              color: 'secondary',
-              textTransform: 'uppercase',
+              cursor: dataset.public ? 'pointer' : 'not-allowed', // Changes cursor to indicate disabled state
+              //opacity: dataset.public ? 1 : 0.5, // Reduces opacity to indicate disabled state
             }}
```

Comment:
@katamartin, i just added functionality for rendering a "disabled" `Open in Data Viewer` when a store is not public. To see this in action, take a look at `Large ensemble pCO2 testbed` feedstock and let me know what you think.

Reply:
This is a good start! We can punt to another PR, but I think this treatment could use a little more thinking.

Without any further messaging, it's not clear why the button is disabled. Because this is a public-facing website, it seems helpful to show a general message that the dataset is not publicly accessible. I think it'd make sense to render this at the top-level for the Zarr store, since the same thing may be applicable for users trying to open in Python?

Also in general, it's helpful to add a disabled style that's not interaction-based (e.g., greying out the button) to signal that the button isn't just broken. This is especially relevant on touch devices where the cursor style does not show up.

Reply:
Great points!! i just opened a new issue with your feedback above. 👍🏽  for addressing this in a separate pull request